readme/testing.md

```diff
@@ -75,13 +75,17 @@ Each testing type can be either executed directly, or via a corresponding Phing
 * `blt tests:behat`
 * `blt tests:phpunit`
 
+To execute a single test (with Phing):
+
+* `blt tests:behat:execute -Dbehat.param=tests/behat/features/Examples.feature`
```

Comment:
It works on my build to run a single test or tests/behat/features

I was looking for a way to run just a single test via blt when writing tests without running everything that a 'blt tests:behat" runs. the -D sets the option for the function and works just like the param.